README.adoc

```diff
@@ -341,9 +354,9 @@ When you visit the URL, look for a table similar to the following example:
 
 image::system_table.png[System table, align="center"]
 
-The table contains three rows, one for each of the running system
+The table contains three rows, one for each of the running `system`
 containers. If you can see the loads updating, you know that your `bff` service
-is successfully receiving messages and broadcasting them as SSE to the front-end client
+is successfully receiving messages and broadcasting them as SSE to the client in the `frontend` service
```

Comment:
don't think you need "for display" here,  the messages themselves aren't displayed